config/techreport.json

```diff
@@ -1118,6 +1124,89 @@
         }
       },
       "description": "Comparison placeholder"
+    },
+    "category": {
+      "id": "category",
+      "title": "Categories",
+      "subtitle": "Technology Report",
+      "config": {
+        "default": {
+          "category": "CMS",
+          "app": ["ALL", "WordPress", "Drupal"],
+          "series": {
+            "breakdown": "app"
+          }
+        },
+        "tech_comparison_summary": {
+          "id": "tech_comparison_summary",
+          "table": {
+            "caption": "Summary",
+            "columns": [
+              {
+                "key": "selectTech",
+                "name": "Select technology",
+                "hiddenName": true,
+                "type": "checkbox"
+              },
+              {
+                "key": "technology",
+                "name": "Tech",
+                "type": "heading"
+              },
+              {
+                "key": "origins",
```

Comment:
When filtering to the top 1k rank, I'm seeing technologies with "null" origin counts

https://technology-report-dot-httparchive.uk.r.appspot.com/reports/techreport/category?tech=&geo=ALL&rank=Top+1k&category=CMS

<img width="468" alt="image" src="https://github.com/user-attachments/assets/9cb661ec-ccbb-49cc-ba3e-3e716f6dbf49" />